### PR TITLE
Telos Cloud Migration

### LINKED ISSUE #718: Prototype Telos Cloud substitute using MetaKeep
# Description

We currently have a Telos Cloud implementation that is about to be discontinued, therefore we need a replacement as urgently as possible.

For that, we will use the MetaKeep service to implement a functional prototype for both blockchains Zero and EVM.

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #718

This PR implements a series of notifications for the users to guide them throw the migration process.

Here's is a demo video

[Migration-from-OreId-to-Metakeep.webm](https://github.com/telosnetwork/telos-wallet/assets/4420760/89f0c454-92c5-43d6-b7ba-7717d243d371)

### CODE REVIEW

File: src/antelope/wallets/authenticators/MetaKeepAuth.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+    async logout(): Promise<void> {
+        this.trace('logout');
+        metakeepCache.setLogged('');
+        return Promise.resolve();
+    }
+
+    async isConnectedTo(chainId: string): Promise<boolean> {
+        this.trace('isConnectedTo', chainId);
+        return true;
```
Comment (ezra-sg) [inline src/antelope/wallets/authenticators/MetaKeepAuth.ts:143]: is it not possible to check chainID with metakeep?

File: src/antelope/wallets/ual/MetakeepUAL.ts [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+
+    /**
+     * @param publicKey     The public key to use for signing.
+     * @param data                The data to be signed.
+     * @param helpText        Help text to explain the need for arbitrary data to be signed.
+     *
+     * @returns                     The signature
+     */
+    signArbitrary = async (): Promise<string> => {
+        throw new Error('cleos does not support signing arbitrary data');
```
Comment (ezra-sg) [inline src/antelope/wallets/ual/MetakeepUAL.ts:442]: should this say `Telos Cloud Login` rather than cleos?

File: src/assets/sicial-icons.zip
Comment (ezra-sg) [inline src/assets/sicial-icons.zip:1]: should this zip file be included?

**Comment by ezra-sg** [comment]
wallet name for Zero is showing up twice:
image

**Comment by ezra-sg** [comment]
when i try to login with a new google account from metakeep, i get infiniload and some console errors
image

**Comment by ezra-sg** [comment]
these are the errors i see when attempting to login telos zero with metakeep
image

File: src/antelope/wallets/authenticators/MetaKeepAuth.ts (hunk 1 above)
Comment (Viterbo) [reply src/antelope/wallets/authenticators/MetaKeepAuth.ts:143]: by default, they support any EOS, ETH, or SOL-compatible blockchains.

This is a good topic for a KT I think.

**Comment by Viterbo** [comment]
I'm closing this PR because is not being used and is getting old.

**Pull request closed without merging**